packages/react-aria/src/index.ts

```diff
@@ -10,7 +10,6 @@
  * governing permissions and limitations under the License.
  */
 
-export {useAccordionItem, useAccordion} from '@react-aria/accordion';
```

Comment:
I don't think we can remove these, this would be breaking

Reply:
I believe these were added after the last release.

Reply:
yep, we just got to it in our audit, these can be removed